demo/src/features/mesh/mod.rs

```diff
@@ -6,8 +6,42 @@ use internal::*;
 use rafx::render_feature_mod_prelude::*;
 rafx::declare_render_feature!(MeshRenderFeature, MESH_FEATURE_INDEX);
 
+mod mesh_wireframe {
+    /// The `Mesh` feature generates submit nodes for drawing wireframes into the `WireframeRenderPhase`
+    /// if the view is registered for both the `WireframeRenderPhase` and the optional `MeshWireframeRenderFeature`.
+    /// The `MeshWireframeRenderFeature` is a NOP if the view is not registered for the `MeshRenderFeature`.
+    use rafx::render_feature_mod_prelude::*;
+    rafx::declare_render_feature!(MeshWireframeRenderFeature, MESH_WIREFRAME_FEATURE_INDEX);
+}
+
+mod mesh_basic {
+    /// The `Mesh` feature generates untextured submit nodes if the `MeshBasicRenderFeature` is registered
+    /// on the view. The `MeshBasicRenderFeature` is a NOP if the view is not registered for the `MeshRenderFeature`.
+    use rafx::render_feature_mod_prelude::*;
+    rafx::declare_render_feature!(MeshBasicRenderFeature, MESH_BASIC_FEATURE_INDEX);
```

Comment:
Maybe it's more clear to call it unlit? (I'm being a little picky with names because we're getting enough things that it may get harder to keep track of them.)

Reply:
This is renamed to `untextured`. The other one is called `unlit` already.